### PR TITLE
Update tutorial-configure-openebs-gke.md

### PR DESCRIPTION
**Pull request by ranjithwingrider**
Signed-off-by: Ranjith Raveendran ranjith.raveendran@openebs.io

Update the file with kubectl output

**What this PR does / why we need it**:
Update output of kubeclt which will hep users to understand more

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
#223 

**Special notes for your reviewer**:

### CODE REVIEW

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 1]
```diff
@@ -83,25 +85,28 @@ Create a admin config context from the config shell:
 
 ```bash
 gcloud container clusters list
-kubectl config set-context demo-openebs03 --cluster=gke_strong-eon-153112_us-central1-a_demo-openebs03 --user=cluster-admin
+root@ubuntu:~$ kubectl config set-context test-admin --cluster=gke_strong-eon-153112_us-central1-a_mycluster_223 --user=cluster-admin
+Context "test-admin" modified.
 ```
 
 The below command will prompt for username/password. Provide username as "admin" and password for the admin can be obtained from : Google Cloud Platform -> Container Engine -> (cluster) -> Show Credentials:
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: The following command...obtained from ...Show Credentials.

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 2]
```diff
@@ -83,25 +85,28 @@ Create a admin config context from the config shell:
...
 ```bash
-kubectl config use-context demo-openebs03
-kubectl config use-context gke_strong-eon-153112_us-central1-a_demo-openebs03
+root@ubuntu:~$ kubectl config use-context test-admin
+Switched to context "test-admin".
+
+root@ubuntu:~$ kubectl config use-context gke_strong-eon-153112_us-central1-a_mycluster_223
+Switched to context "gke_strong-eon-153112_us-central1-a_mycluster_223".
 ```
 **To get the admin priviledge to your cluster(cluster-role admin)**
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: ...your cluster (cluster...

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 3]
```diff
@@ -83,25 +85,28 @@ Create a admin config context from the config shell:
...
 
  grant cluster-admin to your current identity
-`$ kubectl create clusterrolebinding myname-cluster-admin-binding --clusterrole=cluster-admin --user=myname@example.org`
-Clusterrolebinding "myname-cluster-admin-binding" created
-
-
+`$ root@ubuntu:~$ kubectl create clusterrolebinding mycluster_223-cluster-admin-binding --clusterrole=cluster-admin --user=ranjith.raveendran@cloudbyte.com
+clusterrolebinding.rbac.authorization.k8s.io "mycluster_223-cluster-admin-binding" created
+`
 Download the latest OpenEBS Operator Files:
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: files using the following commands.

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 4]
```diff
@@ -112,10 +117,39 @@ cd openebs/k8s
 Apply OpenEBS Operator and add related OpenEBS Storage Classes, that can then be used by developers/apps:
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: ... apps.

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 5]
```diff
@@ -112,10 +117,39 @@ cd openebs/k8s
...
+storageclass.storage.k8s.io "openebs-jupyter" created
+storageclass.storage.k8s.io "openebs-mongodb" created
+storageclass.storage.k8s.io "openebs-cassandra" created
+storageclass.storage.k8s.io "openebs-redis" created
+storageclass.storage.k8s.io "openebs-kafka" created
+storageclass.storage.k8s.io "openebs-zk" created
+storageclass.storage.k8s.io "openebs-es-data-sc" created
 ```
-
 Note: The persistent storage is carved out from the space available on the nodes (default host directory : /var/openebs). There are efforts underway to provide administrator with additional options of consuming the storage (as outlined in "openebs-config.yaml"). These are slated to work hand-in-hand with the local storage manager of the kubernetes that is due in Kubernetes 1.7/1.8.
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: **Note:** ...default host directory: /var/openebs)
"openebs-config.yaml" = *openebs-config.yaml*
with the local Kubernetes storage manager ...

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 6]
```diff
@@ -125,26 +159,69 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
 Get the list of storage classes using the below command. Choose the storage class that best suits your application.
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: ... using the following command.

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 7]
```diff
@@ -125,26 +159,69 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
...
+openebs-percona      openebs.io/provisioner-iscsi                               1m
+openebs-redis        openebs.io/provisioner-iscsi                               1m
+openebs-standalone   openebs.io/provisioner-iscsi                               1m
+openebs-standard     openebs.io/provisioner-iscsi                               5m
+openebs-zk           openebs.io/provisioner-iscsi                               1m
+snapshot-promoter    volumesnapshot.external-storage.k8s.io/snapshot-promoter   5m
+standard (default)   kubernetes.io/gce-pd    
 ```
 
 Some sample yaml files for stateful workoads using OpenEBS are provided in the [openebs/k8s/demo](https://github.com/openebs/openebs/tree/master/k8s/demo)
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: workoads  = workloads

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 8]
```diff
@@ -125,26 +159,69 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
...
 
 ```bash
-kubectl apply -f demo/jupyter/demo-jupyter-openebs.yaml
+root@ubuntu:~/demo_k8/openebs/k8s$ kubectl apply -f demo/jupyter/demo-jupyter-openebs.yaml
+deployment.apps "jupyter-server" created
+persistentvolumeclaim "jupyter-data-vol-claim" created
+service "jupyter-service" created
 ```
 
 The above command will create the following, which can be verified using the corresponding kubectl commands:
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: commands.

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 9]
```diff
@@ -125,26 +159,69 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
...
 
 - Launch a Jupyter Server, with the specified notebook file from github
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: server wth the...GitHub.

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 10]
```diff
@@ -125,26 +159,69 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
...
-  (kubectl get deployments)
+  ```
+  root@ubuntu:~/demo_k8/openebs/k8s$  kubectl get deployments
+  NAME                                            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+  jupyter-server                                  1         1         1            1           2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-ctrl   1         1         1            1           2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-rep    3         3         3            3           2m
+  ```
 - Create an OpenEBS Volume and mounts to the Jupyter Server Pod (/mnt/data)
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: mounts = mount

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 11]
```diff
@@ -125,26 +159,69 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
...
+  ```
+  root@ubuntu:~/demo_k8/openebs/k8s$ kubectl get pods
+  NAME                                                             READY     STATUS              RESTARTS   AGE
+  jupyter-server-68d4ddc48-mml47                                   1/1       Running		       	   0          2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-ctrl-6877d997fb-msnxk   2/2       Running             0          2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-rep-8f94c7b48-gdxbb     1/1       Running             0          2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-rep-8f94c7b48-h5d9p     1/1       Running             0          2m
+  pvc-41964ed3-790a-11e8-a986-42010a80017b-rep-8f94c7b48-skssw     1/1       Running             0          2m
+  ```
+- Expose the Jupyter Server to external world via the http://NodeIP:32424 (NodeIP is any of the worker nodes external IP where application Pod is running)
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: Display the Jupyter server by accessing the http... (Node IP is any worker nodes' external IP where the application pod is running).

File: k8s/hyperconverged/tutorial-configure-openebs-gke.md [hunk 12]
```diff
@@ -125,26 +159,69 @@ All you need to do, to use OpenEBS as persistent storage for your Stateful workl
...
+  ```
+  root@ubuntu:~/demo_k8/openebs/k8s$ kubectl get nodes -o wide
+  NAME                                        STATUS    ROLES     AGE       VERSION        EXTERNAL-IP      OS-IMAGE             KERNEL-   VERSION    CONTAINER-RUNTIME
+  gke-ranjith223-default-pool-314af41d-fk18   Ready     <none>    24m        v1.9.7-gke.3   35.224.182.169   Ubuntu 16.04.4 LTS           4.13.0-1015-gcp   docker://17.3.2
+  gke-ranjith223-default-pool-314af41d-hc03   Ready     <none>    24m        v1.9.7-gke.3   35.188.50.11     Ubuntu 16.04.4 LTS           4.13.0-1015-gcp   docker://17.3.2
+  gke-ranjith223-default-pool-314af41d-lgzf   Ready     <none>    24m       v1.9.7-gke.3   35.192.216.68    Ubuntu 16.04.4 LTS   4.13.0-   1015-gcp   docker://17.3.2
+  ```
 Note:To access the Jupyter Server over the internet, set the firewall rules to allow traffic on port 32424 in you GCP / Networking / Firewalls
```
Comment (mahebbar) [inline k8s/hyperconverged/tutorial-configure-openebs-gke.md]: **Note:** To access...in your GCP/Networking/Firewalls.

**Comment by kmova** [comment]
Closing this in favor of moving this to docs.openebs.io

**Pull request closed without merging**